docs/reference/extensions/django-framework.rst

```diff
@@ -43,6 +54,43 @@ application. In the following example we use it to specify ``libpq-dev``:
         # list required packages or slices for your Django application below.
         - libpq-dev
 
+.. _django-async-deps:
+
+``parts`` > ``django-framework/async-dependencies``
+===================================================
+
+In order to be able to use async Gunicorn workers you need to use
+``django-framework/async-dependencies`` part instead of
+``django-framework/dependencies`` part.
+
+To use this just uncomment the following lines:
+
+.. code-block:: yaml
+
+  parts:
+    django-framework/async-dependencies:
+      python-packages:
+        - gunicorn[gevent]
+
+If your project needs additional debs to run, you can add them to
+``stage-packages`` just like it is done in :ref:`django-framework/dependencies <django-sync-deps>`:
```

Comment:
I would remove the end of the sentence for conciseness, i.e., I would remove this part:
```
just like it is done in :ref:`django-framework/dependencies <django-sync-deps>`
```